shared/components/Choice/Choice.jsx

```diff
@@ -49,8 +49,13 @@ const renderColoredLabel = (color, content) => (
   </ColoredTextProvider>
 )
 
-const renderLabel = (label, feedback, checked, disabled) => {
-  const content = <Text size="medium">{label}</Text>
+const renderLabel = (label, description, feedback, checked, disabled) => {
```

Comment:
Adding the `description` prop would technically make this available for core-checkbox. Should we add a separate story to work on adding description text for core-checkbox?

Reply:
I think there's potential for that, but we should consult design.

Reply:
Good catch, I think it's OOS for this fix. We can always add this as a feature to @tds/core-checkbox in the future.